autoscale/src/main/resources/schema/app/20221110093257_CB-19545_Add_new_entity_to_record_autoscaling_triggers.sql

```diff
@@ -0,0 +1,25 @@
+-- // CB-19545 Add new entity to record autoscaling triggers
+-- Migration SQL that makes the change goes here.
+create sequence if not exists scaling_activity_id_seq;
+
+create table if not exists scaling_activity (
+	id bigint default nextval('scaling_activity_id_seq'::regclass) not null
+	constraint scaling_activity_pkey primary key,
+	activity_crn varchar(255),
+	flow_id varchar(255),
+	start_time timestamp,
```

Comment:
Still missing an index on the start_time, which is used for many of the queries.
Do we need an index on the status as well ? How about for queries which based on the foreign key?

Reply:
created separate indexes as discussed